durablestore/dynamodb/stateitem.go

```diff
@@ -0,0 +1,55 @@
+package dynamodb
+
+import (
+	"fmt"
+
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/reflect/protoreflect"
+	"google.golang.org/protobuf/reflect/protoregistry"
+	"google.golang.org/protobuf/types/known/anypb"
+
+	"github.com/tochemey/ego/v3/egopb"
+)
+
+type StateItem struct {
```

Comment:
can we call this `row`? Also I don't think it needs to be exported.

Reply:
In DynamoDB, a `row` is actually called an `item` according to its terminology. That's why I call it `Item`

Ref: https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.CoreComponents.html

Reply:
I have un-exported `item` struct